rest_framework_simplejwt/locale/zh_Hans/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,113 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2021
+# This file is distributed under the same license as the Simple JWT package.
+# zengqiu <zengqiu@qq.com>, 2021.
+
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-06-23 13:29+0800\n"
+"PO-Revision-Date: 2021-06-23 13:29+080\n"
+"Last-Translator: zengqiu <zengqiu@qq.com>\n"
+"Language: zh_Hans\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+
+#: .\authentication.py:79
```

Comment:
for the lines, please use `rest_framework_simplejwt` instead of `.\` Take a look at other formats.